### DISCUSSION TITLE
Argocd application yaml

### DISCUSSION THREAD

**Discussion by cloudoutloud** [updatecli/updatecli#1886] [General]
Can you use updatecli to look at the `targetRevision` field of a Argocd Application file and bump the version?

For example if you declare helm chart using this method https://argo-cd.readthedocs.io/en/stable/user-guide/helm/

**Comment by olblak** [comment]
Hey @cloudoutloud,

> Can you use updatecli to look at the targetRevision field of a Argocd Application file and bump the version?

If the goal is to do it automatically similarly to helmfile or rancher fleet, not today but it should be straightforward to add and would be a nice addition

Feel free to open an issue with what need to be updated

**Comment by cloudoutloud** [comment]
@olblak

[Issue](https://github.com/updatecli/updatecli/issues/1911) created.

Happy to work on it with some guidance.
